src/content/samples/golang.mdx

```diff
@@ -43,84 +51,55 @@ Log into [Okteto Cloud](https://cloud.okteto.com/#/?origin=docs) and click on th
 
-## Step 2: Create your okteto manifest
-
-To start developing on the Go Sample App you first need to create an okteto manifest.
-With the Go Sample App deployed, run the following command to create your [okteto manifest](/docs/reference/manifest/):
+## Step 2: Activate your development container
 
-```console
-$ okteto init --v1
-```
-
-```console
-This command walks you through creating an okteto manifest.
-It only covers the most common items, and tries to guess sensible defaults.
-See https://okteto.com/docs/reference/manifest for the official documentation about the okteto manifest.
-Use the arrow keys to navigate: ↓ ↑ → ←
-Select the deployment you want to develop:
-  ▸ hello-world
-    Use default values
-```
-
-The `okteto init` command will scan the available deployments in your Kubernetes namespace and ask you to pick one.
-Select the `hello-world` deployment. It's the one we deployed on the previous step.
-
-```console
- ✓  hello-world
- ✓  Deployment 'hello-world' successfully analyzed
- ✓  okteto manifest (okteto.yml) created
- i  Run 'okteto up' to activate your development container
-```
-
-The `okteto init` command creates the following `okteto.yml` file:
+The [dev](http://localhost:8080/docs/reference/manifest/#dev-object-optional) section defines how to activate a development container for the Go Sample App:
 
 ```yaml
-name: hello-world
-image: okteto/golang:1
-command: bash
-securityContext:
-  capabilities:
-    add:
-    - SYS_PTRACE
-volumes:
-  - /go/pkg/
-  - /root/.cache/go-build/
-sync:
-  - .:/usr/src/app
-forward:
-  - 8080:8080
-  - 2345:2345
-```
+dev:
+  hello-world:
+    image: okteto/golang:1
+    command: bash
+    sync:
+      - .:/usr/src/app
+    volumes:
+      - /go
+      - /root/.cache
+    securityContext:
+      capabilities:
+        add:
+          - SYS_PTRACE
+    forward:
+      - 2345:2345
+```
+
+The `hello-world` key matches the name of the hello world Deployment. The meaning of the rest of fields is:
 
-This file defines how to activate a development container for the Go Sample App:
-
-- `name`: the name of the Kubernetes deployment you want to put on development mode.
 - `image`: the image used by the development container. More information on development images [here](/docs/reference/development-environments/).
 - `command`: the start command of the development container.
-- `securityContext`: `SYS_PTRACE` is a capability required by the Go debugger.
-- `volumes`: a list of paths in your development container to be mounted as persistent volumes. For example, this is useful to persist the Go cache.
 - `sync`: the folders that will be synchronized between your local machine and the development container.
-- `forward`: a list of ports to forward from your development container.
+- `volumes`: a list of paths in your development container to be mounted as persistent volumes. For example, this is useful to persist the Go cache.
```

Comment:
```suggestion
- `volumes`: a list of paths in your development container to be mounted as persistent volumes. For example, this can be used to persist the Go cache.
```